cmd/cli/plugin/airgapped-network/interface.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package util defines functions to pull/push/List images
+package main
```

Comment:
This may be moved to a different package.

Reply:
Discussed.